examples/simple_example_dynamic/main.tf

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "access_context_manager_policy" {
+  source      = "../.."
+  parent_id   = var.parent_id
+  policy_name = var.policy_name
+}
+
+module "bridge" {
+  source         = "../../modules/bridge_service_perimeter"
+  policy         = module.access_context_manager_policy.policy_id
+  perimeter_name = "bridge_perimeter_1"
+  description    = "Some description"
+
+  resources = concat(
+    module.regular_service_perimeter_1.shared_resources["all"],
+    module.regular_service_perimeter_2.shared_resources["all"],
+  )
+  resource_keys = ["one", "two", "three"]
```

Comment:
This is slightly confusing as its not apparent `module.regular_service_perimeter_2.shared_resources["all"]` is a list of two elems and hence there are three resource_keys.  Can we add a comment?

Reply:
Switched to using explicit list for resources to make this clearer.